app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -15,6 +18,23 @@ class HomeViewModel
     private val navigationManager: NavigationManager
 ) : ViewModel() {
 
+    private var lastScrollIndex = 0
+    private val _scrollDown = MutableLiveData(false)
+
+    val scrollDown: LiveData<Boolean>
+        get() = _scrollDown.distinctUntilChanged()
+
+    fun updateScrollPosition(newScrollIndex: Int) {
+        if (newScrollIndex == lastScrollIndex) return
+        if (newScrollIndex > lastScrollIndex + 1) {
+            _scrollDown.value = true
+            lastScrollIndex = newScrollIndex
+        } else if (newScrollIndex < lastScrollIndex) {
+            _scrollDown.value = false
+            lastScrollIndex = newScrollIndex
+        }
+    }
```

Comment:
I'm not sure if LiveData has a `scan` operator, but `Flow` has it so by switching to Flow and using it we wouldn't need to have the `var lastScrollIndex`

We already use Flow so I feel like we should stick to the single solution and keep using only this one and we don't need LiveData libraries in the app 🤔